packages/core/src/common/utils/get-ajv-instance.spec.ts

```diff
@@ -116,6 +164,41 @@ describe('getAjvInstance', () => {
           schemaPath: '#/properties/b/type',
         },
       ]);
+
+      const schema6 = {
```

Comment:
Could we add a `// $data` comment before this assignment? I think it could be a nice small improvement.

Reply:
hmm I originally had this here. I got some weird results from `lint:fix` script. It must have removed this.